lib/Dist/Zilla/Dist/Builder.pm

```diff
@@ -776,13 +777,8 @@ does it clean up C<$directory> afterwards.
 sub run_tests_in {
   my ($self, $target, $arg) = @_;
 
-  Carp::croak("you can't test without any TestRunner plugins")
-    unless my @testers = @{ $self->plugins_with(-TestRunner) };
-
-  for my $tester (@testers) {
```

Comment:
Was it intended to remove these lines that call $tester->test for all testing plugins?